HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -0,0 +1,161 @@
+#ifndef HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+#define HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+
+
+#include<cassert>
+#include<cstdint>
+#include<algorithm>
+#ifndef __NVCC__
+#include<atomic>
+#endif
+#ifdef __NVCC__
+#include <cuda_runtime.h>
+#endif
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cudastdAlgorithm.h"
+
+#ifdef __NVCC__
+
+  template<typename Histo>
+  __host__
```

Comment:
I would drop `__host__`, as it is the default anyway.

Reply:
fine with me